tests/components/prometheus/test_init.py

```diff
@@ -97,7 +98,7 @@ async def test_view(hass, hass_client):
     resp = await client.get(prometheus.API_ENDPOINT)
 
     assert resp.status == 200
-    assert resp.headers["content-type"] == "text/plain"
+    assert resp.headers["content-type"] == CONTENT_TYPE_TEXT_PLAIN
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6184085)